src/plotting.py

```diff
@@ -1707,7 +1706,7 @@ def draw_correlation_hist(problem, plot_options):
         varnames = problem.hypernames
     else:
         sc = problem.config.sampler_config
-        varnames = problem.varnames + problem.hypernames + ['like']
+        varnames = list(problem.varnames) + problem.hypernames + ['like']
```

Comment:
let us fix this in the problems.varnames property

Reply:
ok sure, i was not sure if this would pop out as a problem somewhere else than,